int32 camera_id
bool withdraw
---
bool success